Practice2/README.md

```diff
@@ -34,9 +34,18 @@
     * images/
   * styles/
     * base/
+      * all.css
       * base.css
       * reset.css
     * components/
+      * common/
+        * all.css
+        * button.css
+        * heading-default.css
+        * heading-medium.css
+        * heading.css
+        * input-stay-in-touch.css
+      * all.css
       * choose-your-plan.css
       * features.css
       * footer.css
```

Comment:
Should `header` and `footer` be in layout? It belongs to wireframe, right?

Reply:
I don't understand this comment

Reply:
You are listing `header` and `footer` as components, but it's not. `header` and `footer` are parts of layouts. You can search around to get some info about this.

Reply:
Fixed

Reply:
@duylong13 What did you do to these files?

Reply:
I renamed these 2 files 'top' and 'bot'